### PR TITLE
AMD Media Decode - Measure decode time

### PR DESCRIPTION
**Pull request by paveltc**
Fixed issues @asalmanp mentioned in previous pull request.

### CODE REVIEW

File: amd_openvx_extensions/amd_media/decoder.cpp [hunk 1]
```diff
@@ -741,6 +754,11 @@ void CLoomIoMediaDecoder::DecodeLoop(int mediaIndex)
                 return;
             }
             int status = avcodec_receive_frame(videoCodecContext[mediaIndex], frame);
+#if ENABLE_PERF_MEASURE
```
Comment (asalmanp) [inline amd_openvx_extensions/amd_media/decoder.cpp:757]: @paveltc this PR is the same as PR#962. please take a look to my comments for PR#962 and make the requested changes. Thanks.

**Review by asalmanp — CHANGES_REQUESTED** [review]

**Comment by asalmanp** [comment]
> Fixed issues @asalmanp mentioned in previous pull request.

@paveltc I don't see the changes that I requested for PR#962. please review again my comments and apply the changes for this PR.

File: amd_openvx_extensions/amd_media/decoder.cpp (hunk 1 above)
Comment (paveltc) [reply amd_openvx_extensions/amd_media/decoder.cpp:757]: @asalmanp Looking into it. Somehow the older version of the file got added.

**Comment by kiritigowda** [comment]
#964 replaces this PR

**Pull request closed without merging**